routes/projects-route.js

```diff
@@ -26,6 +28,8 @@ projectRouter.use('/characters', characterRouter)
 projectRouter.use('/documents', documentRouter)
 projectRouter.use('/matrices', matrixRouter)
 projectRouter.use('/taxa', taxaRouter)
+projectRouter.use('/users', projectUsersRouter)
+projectRouter.use('/views', mediaViewsRouter)
```

Comment:
I think "views" looks really vague and can mean many different things. Can we make it more clear to be explicitly media-views?